modules/ROOT/pages/java-support.adoc

```diff
@@ -0,0 +1,134 @@
+| 4.9 LTS | February 2025 | 17 | February 2026 | February 2027
+| 4.9 Edge | February 2025 | 17 | June 2025 | October 2025
+|===
+
+[NOTE]
+Along with the end of support for Java 8 with the Mule Runtime 4.9 release in February 2025, Java 11 will also no longer be available. Plan your upgrade path for apps that are running on both Java 8 and Java 11.
+
+
+== Upgrading Mule Apps
+
+Before you upgrade your integration apps or Mule Gateway policies and proxies to Java 17, all extensions, modules, and connectors used within those apps and policies must first be Java 17-compatible. 
+
+== Upgrading Anypoint Connectors and Modules
+
+Anypoint Connectors and modules are certified by MuleSoft to run on Java 17. For information about Java 17 support for each connector and module, see the xref:release-notes::connector/anypoint-connector-release-notes.adoc[Release Notes] for that connector or module. Java compatibility information is also provided on the Exchange tiles for connectors and modules. 
+
+The most commonly used Anypoint Connectors and modules are Java 17-compatible now, and others are being released regularly.
+
+To upgrade Anypoint Connectors:
+
+. Update your Mule app to run on Mule Runtime 4.6 or later. 
+. In Anypoint Studio, update the connectors in your app to the latest version. 
+. Deploy your Mule application:
+  * xref:runtime-fabric::/deploy-to-runtime-fabric.adoc[Deploying Apps to Runtime Fabric]
+  * xref:runtime-manager::cloudhub/deploy-mule-application-task.adoc[Deploying Apps to CloudHub]
+  * xref:hosting::/ch2-deploy.adoc[Deploying Apps to CloudHub 2.0]
+
+== Upgrading Custom Connectors
+
+Custom connectors are any connectors that are not developed and maintained by MuleSoft, including connectors that are built by MuleSoft partners using REST Connect, Mule SDK, or XML SDK.
+
+If you are using a custom connector in your app, you must update your connector to run on Java 17 and MuleSoft Runtime 4.6 and later. After you update your connector using Mule SDK, XML SDK, or REST Connect, you must test your app and deploy it. 
+
+image:java-upgrade-connectors.png[Java upgrade steps for connectors]
+
+[calloutlist]
+. Update your connector using:
+  * xref:mule-sdk/getting-started.adoc[Mule Java SDK]
+  * xref:mule-sdk/xml-sdk.adoc[Mule XML SDK]
+  * xref:exchange/to-deploy-using-rest-connect.adoc[REST Connect]
+. Test your connector using xref:munit/index.adoc[MUnit].
+. Deploy your Mule application:
+  * xref:runtime-fabric::/deploy-to-runtime-fabric.adoc[Deploying Apps to Runtime Fabric]
+  * xref:runtime-manager::cloudhub/deploy-mule-application-task.adoc[Deploying Apps to CloudHub]
+  * xref:hosting::/ch2-deploy.adoc[Deploying Apps to CloudHub 2.0]
+
+== Upgrading REST Connect Generated Connectors
+
+For connectors that are generated from an API specification, all you have to do to make the connector Java 17-compatible is republish the API specification to Exchange. 
+
```

Comment:
If there are Studio docs that mention the J17 changes (like UI or whatever), we may want to link to those docs in a relevant section here

Reply:
Oh but that will be in March

Reply:
Yep. We will update this in March. :-)